pkg/cosign/bundle/bundle.go

```diff
@@ -21,6 +21,9 @@ import "github.com/sigstore/rekor/pkg/generated/models"
 type Bundle struct {
 	SignedEntryTimestamp []byte
 	Payload              RekorPayload
+
+	// RFC3161Entry contains the recorded entry from timestamp authority servers.
+	RFC3161Entry []byte
```

Comment:
the comment should clarify the format, I’d recommend the same comment we have discussed in the protobuf specs